src/FsAutoComplete/CodeFixes/AddMissingWildcardOperator.fsi

```diff
@@ -0,0 +1,12 @@
+module FsAutoComplete.CodeFix.AddMissingWildcardOperator
+
+open FsToolkit.ErrorHandling
+open FsAutoComplete.CodeFix.Navigation
+open FsAutoComplete.CodeFix.Types
+open Ionide.LanguageServerProtocol.Types
+open FsAutoComplete
+open FsAutoComplete.LspHelpers
+
+val title: string
+/// a codefix that adds a missing wildcard pattern to a match case
+val fix: getParseResultsForFile: GetParseResultsForFile -> (CodeActionParams -> Async<Result<Fix list, string>>)
```

Comment:
nit: We have an alias for `(CodeActionParams -> Async<Result<Fix list, string>>)` called `CodeFix` that we should encourage folks to use for consistency's sake.